pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-efs/guide.fr-fr.md

```diff
@@ -0,0 +1,419 @@
+
+It also assumes you have an OVHcloud Enterprise File Storage already available. If you don't, you can [order one in the OVHcloud Control Panel](/links/manager).
+
+You also need to have [Helm](https://docs.helm.sh/) installed on your workstation, please refer to the [How to install Helm on OVHcloud Managed Kubernetes Service](/pages/public_cloud/containers_orchestration/managed_kubernetes/installing-helm) tutorial.
+
+## Instructions
+
+### Step 1 - Creating a partition and granting your Managed Kubernetes Service access to it
+
+Your Enterprise File Storage can expose multiple partitions, and supports a variety of protocols. Each partition is accessible only from a specific range of IPs. We will create one exposing EFS and make it accessible from your Kubernetes worker nodes.
+
+Access the UI for OVHcloud Enterprise File Storage by clicking the `Storage and backups`{.action} then `Enterprise File Storage`{.action} menu in the [Bare Metal Cloud section of the OVHcloud Control Panel](/links/manager)
+
+Click your Enterprise File Storage service, then click the `Volumes`{.action} tab. Click the `Create a volume`{.action} button and create the new Enterprise File Storage volume with the following content:
+
+![Create an EFS partition](images/create-efs-volume.png){.thumbnail}
+
+Here are the parameters you may use to create your volume : 
+
+| Name                | Description                | Required      |
+| ------------------- | -------------------------- | ------------- |
+| Volume name         | Name of the volume         | False         |
+| Volume description  | Description of the volume  | False         |
+| Protocol            | Protocol used to connect   | True          |
+| Volume size         | Size of the volume         | True          |
+
+The volume size needs to be adapted with your needs. For this guide, we define a volume size to 100GiB.
+
+Once your volume is created, clic its ID, then on `Access Control List`{.action}.
+This will let you add your Nodes' Public IPs and/or your Public Cloud Gateway Public IP into the volume's ACLs.
+
+#### Your cluster is installed with Public Network or a private network without using an OVHcloud Internet Gateway or a custom one as your default route
+
+Once the volume is created, we need to allow our Kubernetes nodes to access our newly created partition.
+
+Get our Kubernetes nodes IP:
+
+```bash
+kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="InternalIP")].address }'
+```
+
+```console
+$ kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="InternalIP")].address }'
+51.128.xx.xx 37.59.xx.xx
+```
+
+#### Your cluster is installed with Private Network and a default route via your Private Network (OVHcloud Internet Gateway/OpenStack Router or a custom one)
```

Comment:
Hi Thomas,

Thanks your for contributing.
Since Wednesday Enterprise File Storage is available through vrack directly for region LIM, RBX and GRA.
So should we consider to add a this alternative scenario for mounting volume in k8s directly through vrack to have an enhanced performance experience?

[Here](https://help.ovhcloud.com/csm/en-public-cloud-storage-netapp-network-configuration?id=kb_article_view&sysparm_article=KB0062715) some information are available about the configuration steps.